documentation/Database-Schema-Template.md

```diff
@@ -20,9 +20,9 @@ The `name` is what users of the system will see as the "Application Type" -- e.g
 
 The `is_linear` is used to define the applications flow. If linear it is require each page to be complete before allowing the user to move to the next one. If not the user can go to any page, and only when trying to go to the summary page would be required that all required fields have been completed.
 
-The `start_title` and `start_message` are optional and would define if the application should have a start page or not (when both are Null). In case it is defined the start page is showed when the user clicks on the application in the list or when the user wants to start one.
+The `start_message` is an optional JSON field written using markdown format. If this message is defined then the application show a start page otherwise not. In case it is defined the start page is showed when the user clicks on the application in the list or when the user wants to start a new one.
```

Comment:
Could you define the field something like this?

> The JSON field is an evaluator expression that returns a markdown string. (*Query evaluation not yet implemented*)

Reply:
Sure. Adding this now

Reply:
Ok. Have done something, see if that's a good solution please.